package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "traitify-widgets",
-  "version": "1.3.0",
+  "name": "traitify-widgets-test",
+  "version": "1.3.1-alpha-1",
```

Comment:
I think you can update the name back and bump to 1.4.0. Besides the extra file and the duplication I think it's finished